tests/acceptance/PublishModelCest.php

```diff
@@ -2,15 +2,52 @@
 use Codeception\Util\Locator;
 class PublishModelCest
 {
-	public function i_can_publish_a_model_entry(AcceptanceTester $i)
+	public function _before(\AcceptanceTester $i)
 	{
 		$i->maximizeWindow();
 
 		// First we create a model with fields.
 		$i->loginAsAdmin();
-		$i->haveContentModel('goose', 'geese', 'Geese go honk');
+		$i->haveContentModel('goose', 'gooses', 'gooses go frmh');
```

Comment:
```suggestion
		$i->haveContentModel('goose', 'geese', 'Geese go honk');
```

...to get it back to what it was before... gooses? :laughing: